Src/WitsmlExplorer.Frontend/components/ContentViews/ObjectSearchListView.tsx

```diff
@@ -112,6 +112,51 @@ export const ObjectSearchListView = (): React.ReactElement => {
     }
   };
 
+  const onContextMenuMultipleObjects = async (
+    checkedObjectRows: ObjectSearchRow[],
+    position: MousePosition
+  ) => {
+    const onlyOneObjectType = checkedObjectRows.every(
+      (row) => row.objectType === rows[0].objectType
+    );
+    if (!onlyOneObjectType) {
+      return;
+    }
+    const objectType = checkedObjectRows[0].objectType;
+
+    dispatchOperation({
+      type: OperationType.DisplayContextMenu,
+      payload: {
+        component: (
+          <ContextMenu
+            menuItems={[
+              getBatchModifyMenuItem(
```

Comment:
There seems to be an issue with the frontend UI not updating properly when batch updating logs in the search list view. I think it should be fixed in this component?

Reply:
The search view does not use the well object in navigationContext that the RefreshAction-RefreshHandler functionality relies on. I'm not sure if it's worth it to use time to implement specific handling of the search view, when it only makes a difference if you batch update the names.

Reply:
Created an issue for it.